### PR TITLE
Fixes issues with crons and some local dev workflows

### PR DESCRIPTION
**Pull request by ebenoist**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,16 +9,22 @@ ENVIRONMENT = `cat .env | xargs`
 EXAMPLE_RUN_DEVELOPMENT_FILE = run-development.example.ts
 RUN_DEVELOPMENT_FILE = run-development.ts
 
+all: dist/index.js lib/index.js
+
 dist/index.js: ${SRC} ## build project
 	ncc build src/index.ts
 
+lib/index.js: ${SRC}
```
Comment (stevenweber) [inline Makefile:17]:
```suggestion
# this explicit intermediate rule is unnecessary for building dist/index.js, but is useful when developing so you can import specific source files, see also run-development
lib/index.js: ${SRC} 
```

File: src/input.ts [hunk 2]
```diff
@@ -14,19 +14,13 @@ export interface Input {
 
 export function getInput(): Input {
   return {
-    githubToken: core.getInput('github_token', {
-      required: true,
```
[the remark this answers is not in the document]
Comment (ebenoist) [reply src/input.ts:18]: Yup, I think thats one of the parts I was missing. The way I was testing made this harder given these harder edges as this function is brought in non-lazily to the other modules. Happy to take a look at better options though.

File: src/on_pull_request.ts [hunk 3]
```diff
@@ -7,34 +7,73 @@ import { formatComment } from './github';
...
-    return;
+async function onLabel(octokit: GitHubClient, context: Context, input: Input) {
+  const { actor, repo } = context;
+  const { label } = context.payload;
+
+  const issue = context.payload.issue || context.payload.pull_request;
+  const { number, html_url } = issue;
+
+  core.debug(`label event received for ${number} - ${html_url}`);
+  postMessage(`<${html_url}|#${number}> (${repo.repo}) _*${label.name}*_ by ${actor}`);
```
[the remark this answers is not in the document]
Comment (ebenoist) [reply src/on_pull_request.ts:44]: It was, but I hear your concern as well. I could scope this to console issues? But given that no one else currently uses them I thought it'd be safe, but happy to be conservative as well.

**Comment by ebenoist** [comment]
Ha, merging your code invalidates your review :)

**Comment by ebenoist** [comment]
Catch-22

File: src/on_pull_request.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (stevenweber) [reply src/on_pull_request.ts:44]: Either works for me, just wanted to make sure it wasn't something that was unintentionally lost in a shuffle.

**Comment by stevenweber** [comment]
Ha, unfortunately used to it already.

File: src/on_pull_request.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ebenoist) [reply src/on_pull_request.ts]: Updated— scoped down using a closure for kicks?

**Pull request merged**